vermouth/selectors.py

```diff
@@ -20,31 +20,40 @@
 from .molecule import attributes_match
 
 
-# TODO: Make that list part of the force fields
 PROTEIN_RESIDUES = set(('ALA', 'ARG', 'ASP', 'ASN', 'CYS',
                         'GLU', 'GLN', 'GLY', 'HIS', 'ILE',
                         'LEU', 'LYS', 'MET', 'PHE', 'PRO',
                         'SER', 'THR', 'TRP', 'TYR', 'VAL'))
 
 
-def is_protein(molecule):
+def is_protein(molecule, canonical=False):
     """
-    Return True if all the residues in the molecule are protein residues.
+    Return True if all the residues in `molecule` are protein residues.
 
     The function tests if the residue name of all the atoms in the input
-    molecule are in ``PROTEIN_RESIDUES``.
+    molecule are defined as such in the molecule's force field (under the
+    'protein_resnames' macro). If `canonical` is set to `True`, or if the
```

Comment:
Not a macro at this point, but an attribute.

Reply:
I really meant to say that it should be a macro in the .ff, because otherwise this feature is little documented elsewhere. I'll clarify.